app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughActivityTest.kt

```diff
@@ -173,6 +189,15 @@ class WalkthroughActivityTest {
     }
   }
 
+  @Test
+  fun testWalkthroughActivity_hasCorrectActivityLabel() {
+    val scenario = launch(WalkthroughActivity::class.java)
+    scenario.onActivity { activity ->
```

Comment:
Please write test case similar to `HomeActivityTest` also written in https://github.com/oppia/oppia-android/pull/3850/files

Reply:
thank you. I will check and update

Reply:
@rt4914 Please review , test case updated